http_client.go

```diff
@@ -318,6 +319,14 @@ func (c *HTTPClient) Send(data []byte) (response []byte, err error) {
 		}
 	}
 
+	if (c.config.CloseConnectionOnResponseStatus > 0) {
+		responseStatus, atoiErr := strconv.Atoi(string(payload[9:12]))
```

Comment:
Pls use `proto.Status` https://github.com/buger/gor/blob/master/proto/proto.go#L441 instead, and avoid `Atoi`. It will be smth like `if bytes.Equal(proto.Status(payload), []byte("400"))`